10x/benchmark/10x_cost_analysis.ipynb

```diff
@@ -0,0 +1,295 @@
+       "<matplotlib.figure.Figure at 0x113296e10>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# plot cost\n",
+    "barplot(cost_results_numeric, 'total_cost', 'cost per task ($US)', './cost.png');"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {
+    "collapsed": false,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [
+    {
+     "data": {
+      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAjgAAAFgCAYAAAC2QAPxAAAABHNCSVQICAgIfAhkiAAAAAlwSFlz\nAAALEgAACxIB0t1+/AAAIABJREFUeJzt3XmYZGV5/vHvPSwi+yADyiIzIKCoKDhsggsiCqKABhSC\nSpQEgyjgDmok0ahgRI0aSVCW0RAUFBXFhUVkREAc9v0nsiiIMCogooID9++Pcwpqmp7p6uquPvWe\nvj/X1VfXeU/V9HOmq6qfepfnlW0iIiIi2mRG0wFERERETLYkOBEREdE6SXAiIiKidZLgREREROsk\nwYmIiIjWSYITERERrZMEJyIiIlonCU5ERES0ThKciIiIaJ1lmw5gItZcc03Pnj276TAiIiJiilx6\n6aW/sz1rrPsVneDMnj2bBQsWNB1GRERETBFJt/VyvwxRRUREROskwYmIiIjWSYITERERrZMEJyIi\nIlonCU5ERES0ThKciIiIaJ0kOBEREdE6SXAiIiKidYou9DfdzD78zKZDWKpbj9qt6RAiIiKA9OBE\nRERECyXBiYiIiNZJghMRERGtM7AER9IJku6WdM2I9rdLukHStZI+0dV+hKSbJN0o6eWDiisiIiLa\nb5CTjE8CPg98udMgaUdgD+A5th+UtFbdvhmwD/BMYB3gHEmb2H54gPFFRERESw2sB8f2fOAPI5oP\nAo6y/WB9n7vr9j2Ar9p+0PYtwE3A1oOKLSIiItptqufgbAK8QNLPJJ0vaau6fV3g1133u71uexxJ\nB0paIGnBwoULBxxuRERElGiqE5xlgTWAbYH3AKdK0nj+AdvH2Z5re+6sWbMGEWNEREQUbqoTnNuB\n0125BHgEWBO4A1i/637r1W0RERER4zbVCc63gB0BJG0CLA/8DjgD2EfSEyTNATYGLpni2CIiIqIl\nBraKStIpwIuBNSXdDhwJnACcUC8dfwjY37aBayWdClwHLAIOzgqqiIiI6NfAEhzb+y7h1OuXcP+P\nAh8dVDwRERExfaSScURERLROEpyIiIhonSQ4ERER0TpJcCIiIqJ1kuBERERE6yTBiYiIiNZJghMR\nERGtkwQnIiIiWicJTkRERLROEpyIiIhonSQ4ERER0TpJcCIiIqJ1kuBERERE6yTBiYiIiNZJghMR\nERGtkwQnIiIiWmdgCY6kEyTdLemaUc69S5IlrVkfS9JnJd0k6SpJWw4qroiIiGi/QfbgnATsMrJR\n0vrAy4BfdTXvCmxcfx0IHDvAuCIiIqLlBpbg2J4P/GGUU58G3gu4q20P4MuuXAysLukpg4otIiIi\n2m1K5+BI2gO4w/aVI06tC/y66/j2um20f+NASQskLVi4cOGAIo2IiIiSTVmCI2lF4P3Ahyby79g+\nzvZc23NnzZo1OcFFREREqyw7hT9rI2AOcKUkgPWAyyRtDdwBrN913/XqtoiIiIhxm7IeHNtX217L\n9mzbs6mGoba0/VvgDOCN9WqqbYH7bN85VbFFREREuwxymfgpwEXAppJul3TAUu7+PeBm4Cbgi8Bb\nBxVXREREtN/Ahqhs7zvG+dldtw0cPKhYIiIiYnpJJeOIiIhonSQ4ERER0TpJcCIiIqJ1kuBERERE\n6yTBiYiIiNZJghMRERGtkwQnIiIiWicJTkRERLROEpyIiIhonSQ4ERER0TpJcCIiIqJ1kuBERERE\n6yTBiYiIiNZJghMRERGtkwQnIiIiWicJTkRERLROEpyIiIhonYElOJJOkHS3pGu62v5D0g2SrpL0\nTUmrd507QtJNkm6U9PJBxRURERHtN8genJOAXUa0nQ08y/bmwP8DjgCQtBmwD/DM+jFfkLTMAGOL\niIiIFhtYgmN7PvCHEW1n2V5UH14MrFff3gP4qu0Hbd8C3ARsPajYIiIiot2anIPzZuD79e11gV93\nnbu9bnscSQdKWiBpwcKFCwccYkRERJSokQRH0geARcDJ432s7eNsz7U9d9asWZMfXERERBRv2an+\ngZL+AXglsJNt1813AOt33W29ui0iIiJi3Ka0B0fSLsB7gd1t/7nr1BnAPpKeIGkOsDFwyVTGFhER\nEe0xsB4cSacALwbWlHQ7cCTVqqknAGdLArjY9j/bvlbSqcB1VENXB9t+eFCxRURERLsNLMGxve8o\nzccv5f4fBT46qHgiIiJi+kgl44iIiGidJDgRERHROklwIiIionWS4ERERETrJMGJiIiI1kmCExER\nEa2TBCciIiJaJwlOREREtE4SnIiIiGidJDgRERHROklwIiIionWS4ERERETrJMGJiIiI1kmCExER\nEa2TBCciIiJaJwlOREREtM7AEhxJJ0i6W9I1XW1rSDpb0i/q7zPrdkn6rKSbJF0lactBxRURERHt\nN8genJOAXUa0HQ6ca3tj4Nz6GGBXYOP660Dg2AHGFRERES03sATH9nzgDyOa9wDm1bfnAXt2tX/Z\n
```

Comment:
I feel this notebook is super useful, especially the $244.01 here along with the bar plots.